js/build/shortcode-ui.js

```diff
@@ -206,12 +206,18 @@ Shortcode = Backbone.Model.extend({
 
 		} );
 
+        console.log( attrs.length );
+
 		if ( this.get( 'inner_content' ) ) {
 			content = this.get( 'inner_content' ).get( 'value' );
 		}
 
 		template = "[{{ shortcode }} {{ attributes }}]"
 
+        if( attrs.length < 1 ) {
+            template = "[{{ shortcode }}]"
+        }
```

Comment:
Looks like indentation is off a bit here. Also, can you make this an `if () else {}` statement?